.github/workflows/backport.yml

```diff
@@ -0,0 +1,30 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+    branches:
+      - develop
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
```

Comment:
so we want to run this on all closed PRs? Sometimes they're closed because the author screwed something up and just needs to retry.

Reply:
No, only merged ones, hence L17, but a merged PR is either closed when merged (wtf) or merged and then labelled.

Reply:
To put it alternatively

For all PRs with a base branch of `develop`:
+ If the PR is merged
+ AND one of:
  + the action we're reacting to is `closed` (PR just merged)
  + the action we're reacting to is `labelled` and the label includes `backport`

Reply:
sorry, I'm failing to see where we guard against backporting literally everything here.

Reply:
The action itself, if it finds no labels in the form "backport X" will do nothing